src/Filters/stream_filt.jl

```diff
@@ -178,13 +181,16 @@ end
 
 # Constructor for arbitrary resampling filter (polyphase interpolator w/ intra-phase linear interpolation)
 """
-    FIRFilter(h::Vector, rate::AbstractFloat[, Nϕ::Integer=32])
+    FIRFilter(h::Vector, rate::AbstractFloat, Nϕ::Integer=32, args...)
```

Comment:
```suggestion
    FIRFilter(h::Vector, rate::AbstractFloat, Nϕ::Integer=32)
```
Or did I miss something?